### ISSUE TITLE
Mail on MacOS Ventura 13.02 Issue

### ISSUE THREAD

**Issue by daBee** [mikel/mail#1563]
Tried to update `Mail` as I'm currently showing `version 2.7.1`.  Ran into an error due to a dependency on `net-imap`, which depends on `date` of all things.  

```
An error occurred while installing date (3.3.3), and Bundler cannot continue.
Make sure that `gem install date -v '3.3.3' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  mail was resolved to 2.8.0.1, which depends on
    net-imap was resolved to 0.3.4, which depends on
      date
```

Upon trying that recommendation, I get `ERROR: Failed to build gem native extension.`.  It's also a `default gem`, so I don't know how this is recommending a different version to begin with. So, I'm assuming `date` is valid (`ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [arm64-darwin21]`), so the culprit is unclear.   Thought I'd post here.

**Comment by ahorek** [comment]
it would be useful to share the log of what the actual error is... there's already a discussion about the dependency, see https://github.com/ruby/net-imap/issues/96

**Comment by daBee** [comment]
Here's that log:

```
>>> 9:32:17 user3@myworkstation VP6 ➜ cat /Users/user3/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/extensions/arm64-darwin-21/3.1.0/date-3.3.3/mkmf.log
DYLD_FALLBACK_LIBRARY_PATH=.:/Users/user3/.rbenv/versions/3.1.2/lib "clang -o conftest -I/Users/user3/.rbenv/versions/3.1.2/include/ruby-3.1.0/arm64-darwin21 -I/Users/user3/.rbenv/versions/3.1.2/include/ruby-3.1.0/ruby/backward -I/Users/user3/.rbenv/versions/3.1.2/include/ruby-3.1.0 -I. -I/Users/user3/.rbenv/versions/3.1.2/include  -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE -D_DARWIN_UNLIMITED_SELECT -D_REENTRANT   -fdeclspec -O3 -fno-fast-math -ggdb3 -Wall -Wextra -Wdeprecated-declarations -Wdivision-by-zero -Wimplicit-function-declaration -Wimplicit-int -Wmisleaserver_d-indentation -Wpointer-arith -Wshorten-64-to-32 -Wwrite-strings -Wold-style-definition -Wmissing-noreturn -Wno-cast-function-type -Wno-constant-logical-operand -Wno-long-long -Wno-missing-field-initializers -Wno-overlength-strings -Wno-parentheses-equality -Wno-self-assign -Wno-tautological-compare -Wno-unused-parameter -Wno-unused-value -Wunused-variable -Wextra-tokens -Wundef  -fno-common -pipe conftest.c  -L. -L/Users/user3/.rbenv/versions/3.1.2/lib -L. -L/Users/user3/.rbenv/versions/3.1.2/lib  -fstack-protector-strong -L/usr/local/lib     -lruby.3.1   "
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
checked program was:
/* begin */
1: #include "ruby.h"
2:
3: int main(int argc, char **argv)
4: {
5:   return !!argv[argc];
6: }
/* end */
```

**Comment by daBee** [comment]
FYI, all this is not working still:

```
Using date 3.2.2
Using mail 2.7.1
Using net-imap 0.3.4

cannot load such file -- net/smtp (LoadError)
```

**Comment by j1wilmot** [comment]
This was fixed in #1472. Try using mail 2.8.1 which was just released.
